phpstan.neon

```diff
@@ -1,12 +1,10 @@
 parameters:
   level: max
   paths:
-  - src
-  - tests
+    - %rootDir%/../../../src/
+    - %rootDir%/../../../tests/
   excludes_analyse:
-  - %rootDir%/../../../src/example/*
-  - %rootDir%/../../../src/resources/views/*
-  - %rootDir%/../../../tests/Support/database/*
+    - %rootDir%/../../../tests/Support/database/
```

Comment:
I believe I should switch to `%currentWorkingDir%`; `%rootDir%` doesn't work with Larastan without adjusting which seems odd. currentWorkingDir seems more "stable" as the analyser is always meant to be run from the projects root anyway

Reply:
Okay!